cybsi/api/threat_landscapes/api.py

```diff
@@ -0,0 +1,590 @@
+        Args:
+            landscape_uuid: Threat landscape UUID
+            tag: :attr:`CustomListView.tag` value. Use :meth:`view` to retrieve it.
+            name: new Threat landscape name
+        Raises:
+            :class:`~cybsi.api.error.NotFoundError`: Threat landscape not found.
+            :class:`~cybsi.api.error.ResourceModifiedError`:
+                Threat landscape changed since last request. Update tag and retry.
+        """
+        form = {"name": name}
+        path = f"{_PATH}/{landscape_uuid}"
+        self._connector.do_patch(path=path, json=form, tag=tag)
+
+    def delete(self, landscape_uuid: uuid.UUID) -> None:
+        """
+        Delete a Threat landscape.
+
+        .. versionadded:: 2.14.0
+        Note:
+            Calls `DELETE /threat-landscapes/{landscape_uuid}`
+        Args:
+            landscape_uuid: Threat landscape UUID
+        Raises:
+            :class:`~cybsi.api.error.NotFoundError`: Threat landscape not found.
+        """
+        path = f"{_PATH}/{landscape_uuid}"
+        self._connector.do_delete(path=path)
+
+    def filter_custom_lists(
+        self,
+        landscape_uuid: uuid.UUID,
+        *,
+        cursor: Optional[Cursor] = None,
+        limit: Optional[int] = None,
+    ) -> Page["ThreatLandscapesCustomListView"]:
+        """
+        Get a list of all custom lists related to a Threat landscape.
+
+        .. versionadded:: 2.14.0
+        Note:
+            Calls `GET /threat-landscapes/{landscape_uuid}/custom-lists`
+        Args:
+            landscape_uuid: Landscape UUID
+            cursor: Page cursor
+            limit: Page limit
+        Raises:
+            :class:`~cybsi.api.error.NotFoundError`: Threat landscape not found.
+        """
+        params: Dict[str, Any] = {}
+        if cursor:
+            params["cursor"] = cursor
+        if limit:
+            params["limit"] = limit
+        path = f"{_PATH}/{landscape_uuid}/custom-lists"
+        resp = self._connector.do_get(path=path, params=params)
+        page = Page(self._connector.do_get, resp, ThreatLandscapesCustomListView)
+        return page
+
+    def add_custom_list(
+        self,
+        landscape_uuid: uuid.UUID,
+        custom_list_uuid: uuid.UUID,
+    ) -> None:
+        """
+        Add custom list to a Threat landscape.
+
+        .. versionadded:: 2.14.0
+        Note:
+            Calls `POST /threat-landscapes/{landscape_uuid}/custom-lists`
+        Args:
+            landscape_uuid: Landscape UUID
+            custom_list_uuid: Custom list UUID
+        Raises:
+            :class:`~cybsi.api.error.NotFoundError`: Threat landscape not found.
+            :class:`~cybsi.api.error.SemanticError`:
+        Note:
+            Semantic error codes specific for this method:
+              * :attr:`~cybsi.api.error.SemanticErrorCodes.CustomListNotFound`
+        """
+        params: Dict[str, Any] = {"custom_list_uuid": custom_list_uuid}
```

Comment:
Возможно uuid стоит кастить в str когда передаём в параметрах и в других методах тоже. Кажется были проблемы с этим ранее
```suggestion
        params: Dict[str, Any] = {"custom_list_uuid": str(custom_list_uuid)}
```